web/src/lib/components/forms/library-scan-settings-form.svelte

```diff
@@ -136,9 +153,9 @@
               icon={mdiPencilOutline}
               title={$t('edit_exclusion_pattern')}
               size="16"
-              onclick={() => {
+              onclick={async () => {
                 editExclusionPattern = listIndex;
-                editedExclusionPattern = exclusionPattern;
+                await onEditExclusionPattern();
```

Comment:
In fact you probably don't even need `editExclusionPattern` anymore, right? Can just pass that to the function?